lib/susedistribution.pm

```diff
@@ -317,7 +317,7 @@ sub become_root {
 
     $self->script_sudo('bash', 0);
     type_string "whoami > /dev/$testapi::serialdev\n";
-    wait_serial("root", 10) || die "Root prompt not there";
+    wait_serial('root') || die "Root prompt not there";
```

Comment:
I get the message that 10 seconds is too short , but maybe increasing it to 90 is too much ? what about trying to stop somewhere in a middle at least ? 45 ?